### ISSUE TITLE
Horizontal padding for .postInfo

### ISSUE THREAD

**Issue by ccd0** [ccd0/4chan-x#824]
It doesn't look good when text is pushed right up against the side of the post box.

Currently we have some complicated hacks to avoid the menu button from being flush against the right-hand side of the post box (which looks very ugly) while maintaining reasonable-looking spacing for backlinks placed after it. This could be simplified if we put padding there all the time.

The checkbox's left margin will have to be adjusted.

**Comment by saxamaphone69** [comment]
Is this still a concern?

Here is what it looks like currently:
![passing-before](https://cloud.githubusercontent.com/assets/877480/22860866/593c4bfa-f15e-11e6-81c5-6b6e18fc96ab.png)

After adding `padding-right: 1em; width: auto;` to `.postInfo` (`width` because 4chan sets it to 100%, and probably could be `.5em`) and also giving `.postInfo > .menu-button { padding: 0 5px; }` while removing `width: 18px; height: 15px; text-align: center;` from the current 4chan X styling):
![padding-after](https://cloud.githubusercontent.com/assets/877480/22860867/5a4ee3ea-f15e-11e6-955d-a82251bc9090.png)

**Comment by ccd0** [comment]
@saxamaphone69
I don't think we need that much padding. The first post in your "looks like currently" screenshot looks fine to me. The second could probably use a little more padding.

The "hack" / inconsistency that it would be nice to get rid of is that the spacing between the menu button and its neighbors is done with CSS, whereas the separation between everything else is done with spaces. Problem is that a space can't give you separation between the end of the line and the edge of the box. Thus the desire to put padding there.

All said, it's a very minor concern, and whatever we changed, most of the work would be in testing it to make sure it doesn't make things substantially uglier in some case. There are a lot of cases to consider: no menu, no backlinks, what happens when the line wraps, and probably more I haven't thought of.

**Comment by saxamaphone69** [comment]
Good point. Here is what `.container:not(:empty) { padding-right: .5em; }` looks like [with](https://cloud.githubusercontent.com/assets/877480/22867145/6cfe8b8a-f1d6-11e6-9332-5209c4faf800.PNG) and [without](https://cloud.githubusercontent.com/assets/877480/22867146/702f04d8-f1d6-11e6-904a-3a99550f1891.PNG) the menu, which shows what happens when backlinks go to the next line (and getting that second line to be indented is probably impossible without actual hacks).

Just some food for thought I guess. I think it looks fine, even if backlinks are flush with the edge of the post. I guess something like `.post { padding-right: 0.5em; }` could always be added to someone's Custom CSS if they weren't happy with it (affects all posts regardless of menu or backlinks).

**Comment by ccd0** [comment]
> and getting that second line to be indented is probably impossible without actual hacks

I was considering also adding padding to the left, and adjusting the left margin of the checkbox to compensate. But that would likely add even more complications (e.g. testing with and without the checkboxes hidden).
